tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -54,6 +57,26 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
+		super.genFile(g, outputFileST, fileName);
+		gofmt(fileName); // consider commenting out when debugging
+	}
+
+	private void gofmt(String fileName) {
+		// If the generated source file is for any reason invalid, gofmt will not change the file and generate output on
+		// stderr. The output of stderr will be lost, gofmt may block asynchronously if the output buffer is not
+		// drained, but in that case gofmt will be killed as soon the JVM terminates. In normal operations the gofmt
+		// process will not get killed prematurely and not create a race condition with the lifecycle of the JVM.
+		//
+		// TL;DR: It is safe to fire + forget.
+		try {
+			new ProcessBuilder("gofmt", "-w", "-s", fileName).start();
```

Comment:
Quick question: Where is ProcessBuilder imported from? It's been a while since I've written Java, but my understanding is that it either has to be imported explicitly or implicitly with a *. I don't see either case above.